textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/ListField.java

```diff
@@ -7,15 +7,15 @@
 /**
  * Created by kishorenarendran on 25/04/16.
  */
-public class StringListField implements IField {
-    private final List<String> value;
+public class ListField<T> implements IField {
```

Comment:
My bad, I ended up implementing it anyway! 😛 I'll wait for you to merge into master.